modules/creating-nfd-cr-cli.adoc

```diff
@@ -8,6 +8,11 @@
 
 As a cluster administrator, you can create a `NodeFeatureDiscovery` CR instance by using the {oc-first}.
 
+[NOTE]
+====
+The `spec.operand.image` setting now requires a `-rhel9` image to be defined for use with the 4.16 release of {product-title}. The following example shows the use of `-rhel9` to acquire the correct image.
```

Comment:
@GroceryBoyJr, is this for only the 4.16 release? What about if it is the 4.17 release?

Reply:
Actually, good point, @jab-rh . I discovered this week it's for 4.13+! So I'll edit to support that finding.....

Reply:
So...while this statement is true, it needs to be made....true-er!  :-)

Reply:
Good point, @jab-rh . I have corrected this. Also, I removed the second sentence from the NOTE box, which is referring to the example. No need for that to be in the NOTE box.